### DISCUSSION TITLE
How do you add a webpack plugin?

### DISCUSSION THREAD

**Discussion by bklik** [quasarframework/quasar#13744] [General - Components / Directives / etc]
I found a CSS preload plugin that I want to add to my webpack config:
https://github.com/spnc-omz/PreloadCSSWebpackPlugin

I've looked at the docs for modifying the webpack config, but it doesn't make sense to me as to how I do this:
https://quasar.dev/quasar-cli-webpack/handling-webpack

How do I extend or chain webpack to include this plugin?

**Comment by suleimanas** [comment]
the same as this part:
https://quasar.dev/quasar-cli-webpack/handling-webpack#webpack-v5-compatibility-issues

**Reply by bklik** [reply]
So I should do this?
```
chainWebpack(chain) {
    const PreloadCssPlugin = require('preload-css-webpack-plugin');
    chain.plugin('preload-css').use(PreloadCssPlugin);
},
```

Because that doesn't work. I get `TypeError: compiler.plugin is not a function` when I try to build.
